users-status/README.md

```diff
@@ -0,0 +1,26 @@
+# Firestore user status collection data model
+
+```
+{
+  "id": "String",
+  "userId": "String",
+  "currentStatus": {
+    "state": "OOO | IDLE | ACTIVE",
+    "updatedAt": "TimeStamp",
+    "from": "TimeStamp",
+    "until": "TimeStamp",
+    "message": "String"
+  },
+  "futureStatus": {
+    "state": "OOO | IDLE | ACTIVE",
+    "updatedAt": "TimeStamp",
+    "from": "TimeStamp",
+    "until": "TimeStamp",
+    "message": "String"
+  },
```

Comment:
Can this be 

```
status: {
  current:
  future:
}
```

Reply:
Created the refactor ticket [here](https://github.com/Real-Dev-Squad/website-backend/issues/975)